python/src/numerous/frameworks/dash.py

```diff
@@ -0,0 +1,15 @@
+"""Module for integrating Numerous with Dash."""
+
+from numerous import user_session
+from numerous.frameworks.flask import FlaskCookieGetter
+
+
+class DashCookieGetter(FlaskCookieGetter):
+    pass
+
+
+session = user_session.Session(cg=DashCookieGetter())
+
+
+def get_session() -> user_session.Session:
+    return user_session.Session(cg=DashCookieGetter())
```

Comment:
If we expose `session`, maybe we do not need `get_session`?

Reply:
And maybe we should stick to one, or the other, so the code looks the same for all frameworks.

Reply:
Chose to use get_session function everywhere, since some frameworks could give issues if the session is not updated at each call (multithreaded servers).